src/consensus/consensus.md

```diff
@@ -0,0 +1,293 @@
+
+| Era                                        | Protocol                              | Link                                                                                                                               |
+|:-------------------------------------------|:--------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------|
+| Byron                                      | Ouroboros Classic                     | [Paper](https://iohk.io/en/research/library/papers/ouroboros-a-provably-secure-proof-of-stake-blockchain-protocol/)                |
+| Byron (reimplementation, block forging)    | Ouroboros BFT                         | [Paper](https://iohk.io/en/research/library/papers/ouroboros-bft-a-simple-byzantine-fault-tolerant-consensus-protocol/)            |
+| Byron (reimplementation, block processing) | Ouroboros PBFT                        | Section 4 of the [Byron spec](https://github.com/intersectmbo/cardano-ledger/releases/latest/download/byron-blockchain.pdf)        |
+| Shelley                                    | Ouroboros TPraos (Transitional Praos) | Section 12 of the [Shelley spec](https://github.com/intersectmbo/cardano-ledger/releases/latest/download/shelley-ledger.pdf)       |
+| Allegra                                    | Ouroboros TPraos                      |                                                                                                                                    |
+| Mary                                       | Ouroboros TPraos                      |                                                                                                                                    |
+| Alonzo                                     | Ouroboros TPraos                      |                                                                                                                                    |
+| Babbage                                    | Ouroboros Praos                       | [Paper](https://iohk.io/en/research/library/papers/ouroboros-praos-an-adaptively-secure-semi-synchronous-proof-of-stake-protocol/) |
+| Conway                                     | Ouroboros Praos                       |                                                                                                                                    |
+
+Each of these protocols defines how to fulfill the responsibilities above. Regarding
+validity of blocks, the Consensus layer can remain oblivious to the details on validation
+and rely on the Ledger layer to make such judgment, based on the particular era in effect.
+
+## Selecting chains
+
+_Chain selection_ is the mechanism, specified in the Protocol associated to the current
+era in effect in the network, through which nodes acquire and diffuse the best chain in
+the network. It is important to note that **all nodes** participating in the network are in
```

Comment:
Perhaps would be interesting to outline that this implies even non block forging node contribute to the security of the network?

Reply:
Done :+1: